Azure/sonic-buildimage
# BRCM Disable ACL Drop counted towards interface RX_DRP counters

#### Why I did it
For all BRCM ACL Drops there is a way not to count these packet drops towards the interface RX_DRP counters by enabling a new SOC property. This SOC property is being enabled on the chosens BRCM HW SKUs config.bcm files.

#### How to verify it
Send packet that is expected to be drop and check that it is not being counted towards interface RX_DRP counter.
Also tested on S6100, 7260, and TD3 WARM Reboot scenario where this SOC property was not present before the WARM Reboot but become enabled after WARM Reboot.

#### Which release branch to backport (provide reason below if selected)


Although this change is needed also for the 202012 branch, but the change is not straightforward to cherry pick into that branch so a separate PR will be raised for 202012 branch,

#### Description for the changelog
Add SOC property to not count ACL Drops towards interface RX_DRP for All BRCM HW SKUs.

#### A picture of a cute animal (not mandatory but encouraged)
